### PR TITLE
Sa 477

### PR DESCRIPTION
**Pull request by gwalls**

### CODE REVIEW

File: content/api/events-ingest.apib [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+## Ingest [/v1/ingest]
+
+### Send Events [POST /v1/ingest/events]
+
+Post a new-line delimited list of JSON events.
+
++ Data Structure
+    + msys (object, required) - The event wrapper
+        + [event grouping] (object, required) - Designates the event grouping <br />One of: `message_event`, `track_event`, `gen_event`, `adaptive_event`, `unsubscribe_event`, `relay_event`, `ab_test_event`
```
Comment (kleids) [inline content/api/events-ingest.apib]: screen shot 2019-02-22 at 11 09 27 am

Comment (kleids) [reply content/api/events-ingest.apib]: What do brackets mean? `[event grouping]`

If object, then why does transport method get one?

Comment (kleids) [reply content/api/events-ingest.apib]: Also all of the grouping types are listed, should we list all of the event types too?

File: content/api/events-ingest.apib [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+        Content-Encoding:gzip
+        Content-Type:application/x-ndjson
+
++ Response
+    
+        {
+            "results": {
+                "id": "b2013586-c13f-4da4-baf4-11b9d8e40c50"
```
Comment (kleids) [inline content/api/events-ingest.apib]: Do we need to mention or explain what the ID is on the 200 success?

File: content/api/events-ingest.apib (hunk 1 above)
Comment (gwalls) [reply content/api/events-ingest.apib]: I used brackets to signify "an object key is needed here, but it's one of the following..." . I'm not sure how to signify an object key without giving the exact name of the key, like `type`.

The grouping types _have_ to be one of those listed; however, the value of `type` is not currently validated - we're just checking that it's there. Is there a definitive list of event types? Or I could list some examples?

File: content/api/events-ingest.apib (hunk 2 above)
Comment (gwalls) [reply content/api/events-ingest.apib]: I'm not sure I know what the ID is lol... Can it be used to reference something?

File: content/api/events-ingest.apib (hunk 1 above)
Comment (gwalls) [reply content/api/events-ingest.apib]: Ok, looks like I want Italics and not brackets ref: https://github.com/apiaryio/mson#variable-property-name

**Comment by gwalls** [comment]
closing to hopefully resolve issues

**Pull request closed without merging**